interpretation/ww_head.py

```diff
@@ -113,18 +114,38 @@ def process(message, isGameMaster = False):
         '''add'''
         # This command allows users to add users to a conspiracy.
         # This command will not trigger if the user doesn't own the conspiracy channel.
-        if is_command(message,['add']):
-            # TODO
-            return todo()
+        if is_command(message,['add'], members_to_add=[], channel_id):
```

Comment:
You should only insert what you need when DEFINING the function, not when you're calling a function. Right now you're trying to give the is_command() function parameters that it doesn't want.

Reply:
I think he was trying to specify arguments to the command

Reply:
(Which it still isn't clear how to do)

Reply:
I'm kinda confused now